week-3/implement/get-card-value.js

```diff
@@ -29,3 +29,46 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(str) {
```

Comment:
Be careful of the acceptance criteria on line 9. `str` might be a value such as `A♠`, which contains both the card suit _and_ value